src/cclib/parser/daltonparser.py

```diff
@@ -737,20 +754,36 @@ def extract(self, inputfile, line):
         # @    Final gradient norm:           0.000003746706
         # ...
         #
+        if "Final HF energy" in line and not (hasattr(self, "mpenergies") or hasattr(self, "ccenergies")):
+            self.metadata["theory"] = "HF"
```

Comment:
I would prefer to have both in cclib, since MP2 can follow other things than HF.